src/quo/animated.cljs

```diff
@@ -256,12 +263,13 @@
               (set position (add offset val))])
       (cond* (and* (eq state (:end gh/states))
                    (not* animation-over))
-             [(set position (re-timing
-                             {:clock    c
-                              :easing   easing
-                              :duration duration
-                              :from     position
-                              :to       to}))
+             [(set position
+                   (re-timing
+                    {:clock    c
+                     :easing   easing
+                     :duration duration
+                     :from     position
+                     :to       to}))
```

Comment:
This looks like unnecessary change to me

Reply:
This requires renaming `set` to things like `set!` and treat it as a `:arg2` fn like `clojure.core/reset!`